pootle/apps/pootle_store/views.py

```diff
@@ -515,6 +518,36 @@ def get_stats(request, *args, **kwargs):
 
 
 @ajax_required
+@get_path_obj
+@permission_required('view')
+@get_resource
+def get_contributors(request, *args, **kwargs):
```

Comment:
If you are addding new `views` please use the view classes. Im desperately trying to get rid of get_path_obj and get_resource, and the view will not be pluggable